### PR TITLE
UINV-209 Show the possible duplicate modal

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
https://issues.folio.org/browse/UINV-209

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/invoices/InvoiceForm/InvoiceFormContainer.js [hunk 1]
```diff
@@ -73,18 +81,25 @@ function InvoiceFormContainer({
 
     if (!forceSaveValues) {
       const { vendorInvoiceNo, invoiceDate, vendorId } = formValues;
+      const date = id ? moment.utc(invoiceDate).format(DATE_FORMAT) : invoiceDate;
```
Comment (aliaksei-chumakou) [inline src/invoices/InvoiceForm/InvoiceFormContainer.js:84]: this line looks not clear the purpose. Does it mean that one var contains different data?

Comment (AndreiShumski2) [reply src/invoices/InvoiceForm/InvoiceFormContainer.js:84]: Date from `DatePicker` comes in `YYYY-MM-DD` (2021-02-03), but then it converts on BE to `DateTime` format `2021-02-03T00:00:00.000+0000` and it should be converted back for correct filtering

**Review by axelhunn — APPROVED** [review]

File: src/invoices/InvoiceForm/InvoiceFormContainer.js (hunk 1 above)
Comment (NikitaSedyx) [reply src/invoices/InvoiceForm/InvoiceFormContainer.js:84]: why format of the same field is different?

File: src/invoices/InvoiceForm/InvoiceFormContainer.js [hunk 2]
```diff
@@ -124,6 +139,30 @@ function InvoiceFormContainer({
     links: invoiceDocuments.filter(invoiceDocument => invoiceDocument.url),
   };
 
+  const modalFooter = (
```
Comment (NikitaSedyx) [inline src/invoices/InvoiceForm/InvoiceFormContainer.js:142]: will it be possible to create a modal in separate component?

File: src/invoices/InvoiceForm/InvoiceFormContainer.js (hunk 1 above)
Comment (NikitaSedyx) [reply src/invoices/InvoiceForm/InvoiceFormContainer.js:84]: IMO query should work with `2021-02-03T00:00:00.000+0000` as well

Comment (AndreiShumski2) [reply src/invoices/InvoiceForm/InvoiceFormContainer.js]: I've checked. Unfortunately, it doesn't.

**Review by NikitaSedyx — APPROVED** [review]

**Review by aliaksei-chumakou — APPROVED** [review]

**Pull request merged**